.travis.yml

```diff
@@ -19,8 +19,10 @@ jobs:
   include:
     - stage: "Unit Test"
       name: "Unit Test"
+      before_script:
+        - export GO111MODULE=on
       script:
-        - go get -u github.com/golang/dep/cmd/dep
+        - go mod tidy
```

Comment:
Perhaps this should be `go mod vendor`?

BTW, In Keycloak Operator we pushed the whole vendor directory. This made Travis builds blazing fast.

Reply:
Will change this for travis build, however the build time less than test several times

Reply:
Our tests are not yet that fast (we need to implement mocking), so speed of adding vendor will probably be not so noticeable cos tests are the primary time consumer at the moment.